drivers/hmis_external_apis/lib/tasks/import.rake

```diff
@@ -0,0 +1,12 @@
+namespace :import do
+  task remote_data: [
+    'driver:hmis_external_apis:import:ac_projects',
+    'eto:import:demographics_and_touch_points',
+  ]
```

Comment:
Can we move this task out of the driver?  Makes me a little nervous to have the ETO rake task (scheduled task) dependent on the driver for HMIS data.  If we just have a simple `import:remote_data` task a the app level, it can have an array of task to call, and the driver tasks can be behind `if RailsDrivers.loaded.include?(:hmis_external_apis)`